client/src/variable.scss

```
@@ -1,5 +1,5 @@
 // Variable für Font-Family's:
-$ff-roboto: "Roboto", Roboto;
+$ff-roboto: "Roboto";
```

Bestimmte bitte einen Rückfall-Wert falls Robot nicht definiert ist.
```suggestion
$ff-roboto: "Roboto", Arial, sans-serif;
```

---

done!